client/ipa-getkeytab.c

```diff
@@ -984,7 +985,7 @@ int main(int argc, const char *argv[])
 	krb5_context krbctx;
 	krb5_ccache ccache;
 	krb5_principal uprinc = NULL;
-	krb5_principal sprinc;
+	krb5_principal sprinc = NULL;
```

Comment:
Seems unnecessary, but doesn't hurt.

Reply:
Yeah, this one wasn't called out by the analyzer I just happened to notice it.